### PR TITLE
Bridge slaves selection stays persistent when switching tabs

### PR DESCRIPTION
**Pull request by mchf**
bnc#846201

### CODE REVIEW

File: src/include/network/lan/address.rb [hunk 1]
```diff
@@ -446,6 +446,24 @@ def StoreAddrWidget(key, event)
       nil
     end
 
+    # Initializes widget (BRIDGE_PORTS) which contains list of devices available
+    # for enslaving in a brige.
+    #
+    # @param [String] key	id of the widget
+    def InitBridge(key)
+      br_ports = (@settings["BRIDGE_PORTS"] || "").split
+      br_ports ||= (NetworkInterfaces.Current["BRIDGE_PORTS"] || "").split
```
Comment (jreidinger) [inline src/include/network/lan/address.rb]: this never happen as `"".split` is `[]` so or is always `true`

Comment (mchf) [inline src/include/network/lan/address.rb]: ok, i'll improve the copy paste fix ;-)

**Comment by jreidinger** [comment]
LGTM

**Comment by mchf** [comment]
Thanks for review

**Pull request merged**